build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -33,6 +33,7 @@
     "adreno-730": "gpu-adreno",
     "mali-g77": "gpu-mali-valhall",
     "mali-g78": "gpu-mali-valhall",
+    "tesla-v100-sxm2-16gb": "gpu-cuda-sm_70",
```

Comment:
This is great, would be nice to get a `a100` as well when possible.

Reply:
It will be added in the follow-up changes : )